Source/WebKit/UIProcess/Cocoa/AuxiliaryProcessProxyCocoa.mm

```diff
@@ -155,4 +155,16 @@
 }
 #endif
 
+#if USE(EXTENSIONKIT)
+bool AuxiliaryProcessProxy::requestRemoteProcessTermination()
+{
+    if (auto process = extensionProcess()) {
+        process->invalidate();
+        return true;
+    }
+    RELEASE_LOG_ERROR(Process, "Extension process was not invalidated");
```

Comment:
Please follow the format of our usual logging. It needs to mentioned the class and method name the logging is called from.

Reply:
Ah, good point, fixed in latest patch.